src/index.scss

```diff
@@ -30,346 +37,268 @@ $mobile-width: 750px;
-		height: 14.1vw;
-		font-size: 13vw;
+		max-height: 100vh;
+		height: 100vh;
 	}
 }
-#hackathon-description {
-	height: 43px;
-	font-size: 34px;
-	font-weight: 700;
-	color: $blue;
 
-	@include mobile {
-		height: 8.2vw;
-		font-size: 6.3vw;
-	}
-}
-#hackathon-info {
-	height: 25px;
-	font-size: 17px;
-	font-weight: 500;
-	color: $blue;
+#general {
+	background-color: $light-blue;
+	
+	background-image: url($assets+'shelf.svg'), url($assets+'desk.svg');
+	background-repeat: no-repeat, no-repeat;
+	background-size: 100vw, 85vw;
+	background-position: 0 130%, 24.2vw 75%;
 
 	@include mobile {
-		height: 4vw;
-		font-size: 3.2vw;
+		background-size: 100vw, 120vw;
+		background-position: 0 100%, -6vw 100%;
 	}
-}
 
-.mailchimp-container {
-	height: 50px;
-	width: 450px;
-	margin: 20px auto;
-	
-	border-style: solid;
-	border-width: 1px;
-	border-color: $blue;
-	border-radius: 25px;
+	#wire {
+		position: absolute;
 
-	@include mobile {
-		width: 80vw;
+		left: 60.7vw;
 
-		// On mobile, only enforce top margin
-		margin: 1vh auto auto auto;
+		width: 1.5vw;
+		height: calc(100% - 91vw);
 
-		height: 12vw;
-		border-radius: 6vw;
-	}
-}
-.mailchimp-input {
-	flex: 50%;
-	height: 30px;
-	margin-left: 10px;
-
-	text-align: center;
-	border-style: none;
-
-	outline: none;
-	font-family: inherit;
-	font-size: 17px;
-	font-weight: 500;
-
-	&::placeholder { /* this refers to the input placeholder text (enter your email...)*/
-		color: $blue;
-		transition: color 0.4s;
-	}
-	&:focus::placeholder {
-		color: transparent;
+		background-color: $dark-blue;
 	}
 
-	@include mobile {
-		height: 7vw;
-		font-size: 3.25vw;
-	}
-}
-.mailchimp-button {
-	height: 40px;
-	width: 40px;
-
-	margin-right: 5px;
-
-	border-style: solid;
-	border-width: 1px;
-	border-color: $blue;
-	border-radius: 20px;
-
-	background-color: white;
-	background-image: url(./input-arrow.svg);
-	background-repeat: no-repeat;
-	background-position: 4.5px;
-	background-size: 25px;
+	#general-content {
+		position: absolute;
+		display: inline-block;
+		
+		margin-left: 7.5vw;
+		margin-top: 16vw;
 
-	outline: none;
-	transition: 0.1s;
+		text-align: center;
 
-	@include desktop {
-		&:hover {
-			background-position: 6.5px;
+		#hackathon-name {
+			height: 4.8vw;
+			font-size: 4.5vw;
+			font-weight: 700;
+			color: white;
+		
+			@include mobile {
+				height: 14.1vw;
+				font-size: 13vw;
+			}
 		}
-	}
-
-	@include mobile {
-		height: 9vw;
-		width: 9vw;
-		border-radius: 4.5vw;
-		margin-right: 1.5vw;
-		background-position: 1.5vw;
-		background-size: 5vw;
-	}
-}
+		#hackathon-description {
+			height: 2.8vw;
+			font-size: 2.15vw;
+			font-weight: 700;
+			color: white;
+		
+			@include mobile {
+				height: 8.2vw;
+				font-size: 6.3vw;
+			}
+		}
+		#hackathon-info {
+			height: 1.1vw;
+			font-size: 1.06vw;
+			font-weight: 500;
+			color: white;
+		
+			@include mobile {
+				height: 5vw;
+				font-size: 3.2vw;
+			}
+		}
+		.mailchimp-container {
+			height: 3vw; // 50
```

Comment:
Why are px values commented if we can just use the px values? 
As a note the scaling is quite small on tablet: 
<img width="911" alt="Screen Shot 2020-11-01 at 2 18 52 AM" src="https://user-images.githubusercontent.com/2782749/97797138-97519180-1be8-11eb-9e61-6032dac4e6d1.png">

Reply:
I'll bring this up on design sync; out of scope for this PR

Reply:
Those are the old values that I briefly commented to keep track of proportions, forgot to remove them since I made some changes to the proportions